docs/supported-technologies.asciidoc

```diff
@@ -285,7 +285,7 @@ same trace.
 side, the agent reads the context from the Message property through `javax.jms.MessageConsumer#receive`,
 `javax.jms.MessageConsumer#receiveNoWait`, `javax.jms.JMSConsumer#receive`, `javax.jms.JMSConsumer#receiveNoWait` or
 `javax.jms.MessageListener#onMessage` and uses it for enabling distributed tracing.
-|1.7.0 - Incubating (off by default). Use <<config-disable-instrumentations>> to enable
+|1.7.0 - (incubating - off by default). On by default since 1.13.0.
```

Comment:
I would better write it in the opposite order: `Enabled by default since 1.13.0, added in incubating in 1.7.0 `